style.css

```diff
@@ -338,6 +346,7 @@ ul {
 
   .contact-form {
     margin-top: 80px;
+    border-radius: 70px 12px 12px 12px;
   }
```

Comment:
- [ ] As this is a first review, I'll have to ask you to layout the content of the form using flexbox based on requirements.
- Instructions:
![image](https://user-images.githubusercontent.com/50990925/220568150-63e0c3be-bdb8-4fba-9d64-8abc9697f92a.png)

Reply:
Fixed.